plugins/env/app/assets/javascripts/env/application.js

```diff
@@ -92,4 +117,16 @@
       });
     });
   });
+
+  // Display preview link dynamically when env group dropdown changes
+  $(document).ready(function() {
+    var $inputs = $(".env_group_inputs");
+
+    if (!$inputs) { return; }
```

Comment:
it's never false ... can check `.length` though ... or just delete this line since the .each works